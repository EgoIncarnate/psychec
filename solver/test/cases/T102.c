static T1 const a1[] = { { 1, x1 } };

void f() {}
