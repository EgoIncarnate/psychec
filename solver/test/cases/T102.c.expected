typedef  struct __T1__ {int member_0;uintptr_t member_1;} const T1 ;
#define  x1 128
